hpx/util/unwrapped.hpp

```diff
@@ -522,6 +580,7 @@ namespace hpx { namespace util
         return unwrapped(unwrapped(std::forward<Future>(f)));
     }
 
+    /// \copydoc unwrapped2
```

Comment:
This probably will not be sufficient to make doxygen generate the docs for this function.